### ISSUE TITLE
Unable to connect to PostgreSQL server: FATAL:  the database system is starting up

### ISSUE THREAD

**Issue by maxze** [mediagis/nominatim-docker#29]
Hello,

I have this issue where, after the container was started, I get the following error message when I go to "localhost:8080/search.php". According to "service postgresql status" inside the running container, everything's online (9.3/main (port 5432): online). 

Here's what I see when accessing search.php.

> Internal Server Error
> 
> Nominatim has encountered an internal error while accessing the database. This may happen because the database is broken or because of a bug in the software. If you think it is a bug, feel free to report it over on Github. Please include the URL that caused the problem and the complete error details below.
> 
> Message: Failed to establish database connection
> 
> SQL Error: DB Error: connect failed
> 
> Details:
> 
>  [nativecode=pg_connect(): Unable to connect to PostgreSQL server: FATAL:  the database system is starting up] ** pgsql://@/nominatim 

I also checked the log files and they show the following:

> ==> /var/log/apache2/access.log <==
> 127.0.0.1 - - [22/Nov/2017:05:47:53 +0000] "OPTIONS * HTTP/1.0" 200 125 "-" "Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.22 (internal dummy connection)"
> 127.0.0.1 - - [22/Nov/2017:05:47:53 +0000] "OPTIONS * HTTP/1.0" 200 125 "-" "Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.22 (internal dummy connection)"
> 127.0.0.1 - - [22/Nov/2017:05:47:53 +0000] "OPTIONS * HTTP/1.0" 200 125 "-" "Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.22 (internal dummy connection)"
> 127.0.0.1 - - [22/Nov/2017:05:47:53 +0000] "OPTIONS * HTTP/1.0" 200 125 "-" "Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.22 (internal dummy connection)"
> 127.0.0.1 - - [22/Nov/2017:05:47:53 +0000] "OPTIONS * HTTP/1.0" 200 125 "-" "Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.22 (internal dummy connection)"
> 
> ==> /var/log/apache2/error.log <==
> [Wed Nov 22 05:47:52.919962 2017] [mpm_prefork:notice] [pid 23] AH00163: Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.22 configured -- resuming normal operations
> [Wed Nov 22 05:47:52.920287 2017] [core:notice] [pid 23] AH00094: Command line: '/usr/sbin/apache2'
> [Wed Nov 22 05:47:53.943417 2017] [mpm_prefork:notice] [pid 23] AH00171: Graceful restart requested, doing restart
> 
> ==> /var/log/apache2/other_vhosts_access.log <==
> AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.3. Set the 'ServerName' directive globally to suppress this message
> 
> ==> /var/log/apache2/error.log <==
> AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.3. Set the 'ServerName' directive globally to suppress this message
> [Wed Nov 22 07:17:39.541005 2017] [core:warn] [pid 161] AH00098: pid file /var/run/apache2/apache2.pid overwritten -- Unclean shutdown of previous Apache run?
> [Wed Nov 22 07:17:39.544846 2017] [mpm_prefork:notice] [pid 161] AH00163: Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.22 configured -- resuming normal operations
> [Wed Nov 22 07:17:39.544876 2017] [core:notice] [pid 161] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
> [Wed Nov 22 07:17:52.724268 2017] [authz_core:debug] [pid 162] mod_authz_core.c(802): [client 192.168.100.232:54833] AH01626: authorization result of Require all granted: granted
> [Wed Nov 22 07:17:52.724334 2017] [authz_core:debug] [pid 162] mod_authz_core.c(802): [client 192.168.100.232:54833] AH01626: authorization result of <RequireAny>: granted

Anything I'm doing wrong? I had a working container running in the past and I don't remember doing anything differently.

**Comment by maxze** [comment]
UPDATE: After waiting for a while, the only thing I saw was an empty page (on search.php). Somewhere in the logs I found this line:

> "PHP Fatal error:  Class 'Transliterator' not found in /app/git/lib/Geocode.php on line 53"

I found that other users of Nominatim have had the [same error message](https://github.com/openstreetmap/Nominatim/issues/747). The solution for this is to also [install "php-intl"](https://github.com/openstreetmap/Nominatim/issues/747#issuecomment-306743506). I've manually installed php5-intl inside the container and restarted Apache. Now everything works fine.

However, new builds need php5-intl to be installed so it should be added to the Dockerfile.
